.github/workflows/test.yml

```diff
@@ -53,9 +53,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        test: [bitcoin, monero, swap]
```

Comment:
The timeout-minutes should be significantly reduced again. I think we can go back to the old 45 min limit.

Reply:
Done 9ecacfbae4a48c5dd20c544274f9b7849c117eab